app/assets/javascripts/shopify_app/app_bridge_redirect.js

```diff
@@ -0,0 +1,18 @@
+(function() {
+  function appBridgeRedirect(url) {
+    var AppBridge = window['app-bridge'];
+    var createApp = AppBridge.default;
+    var Redirect = AppBridge.actions.Redirect;
+    var app = createApp({
+      apiKey: window.apiKey,
+      shopOrigin: window.shopOrigin.replace(/^https:\/\//, ''),
+    });
```

Comment:
It looks like we initialize AppBridge here: https://github.com/Shopify/shopify_app/blob/master/lib/generators/shopify_app/install/templates/shopify_app.js

Maybe we can just rely on `window.app` here?

Reply:
Ah, I think I'm wrong and that js isn't getting loaded, and also, it depends on the api key and shop origin params being set in a different place. It looks like the lib has two different patterns for doing this right now. I don't know if now is the right time to fix it, but it's tempting.

Reply:
Using `script` tag in the `head` is more reliable I think as it doesn't have to wait until DOMLoaded.